mrblib/aws-s3.rb

```diff
@@ -1,34 +1,37 @@
+class Module
+  def self.const_get(const)
+    Object.const_get(const)
+  end
+end
+
 module AWS
   class S3
     EMPTY_STRING_SHA256 = Digest::SHA256::hexdigest('')
 
-    def initialize(access_key, secret_key, security_token = nil, region = nil, endpoint = nil)
+    def initialize(access_key, secret_key, security_token = nil, region = nil)
```

Comment:
I know there is too many parameters now but I would like this kind of fix: https://github.com/iij/mruby-aws-s3/issues/7